pandas/tests/reshape/test_reshape.py

```diff
@@ -45,7 +45,7 @@ def test_basic(self, sparse, dtype):
             dtype=self.effective_dtype(dtype),
         )
         if sparse:
-            expected = expected.apply(pd.SparseArray, fill_value=0.0)
+            expected = expected.apply(pd.arrays.SparseArray, fill_value=0.0)
```

Comment:
ok, but i'd still prefer just using SparseArray and importing it at the top, a followup for this would be ok.